webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+
+// When running in docker, we install libraries into /tools/node_modules
+// so that we don't overwrite the user's local kuma/node_modules directory
+// with linux versions of modules (like the binary newrelic module).
+// To make this work, the docker container sets the NODE_PATH environment
+// variable so that code can be found in /tools/node_modules.
+// But we also have to convince webpack and babel to use the /tools
+// directory when we're running in docker, so this nodePath constant
+// is the absolute path to the node_modules director we want to use.
+// We set it on resolve.modules below, and prepend it to the name of
+// all the babel plugins so we're sure we're gettting precisely the
+// file we want both locally (NODE_PATH not set) and in docker.
+//
+// TODO: there ought to be a better way to do this.
+//
+const nodePath = process.env.NODE_PATH || path.join(__dirname, 'node_modules');
+
+module.exports = {
+    mode: "production",  // Or switch to "development"
```

Comment:
This could be loaded from the ``process.env`` as well 😁 . Maybe ``webpack --mode=development`` works too, https://webpack.js.org/concepts/mode/